cryptoki/src/object.rs

```diff
@@ -518,6 +521,8 @@ pub enum Attribute {
     Value(Vec<u8>),
     /// Length in bytes of the value
     ValueLen(Ulong),
+    /// Vendor defined value
+    VendorDefined((CK_ATTRIBUTE_TYPE, Vec<u8>)),
```

Comment:
Why do we need the `CK_ATTRIBUTE_TYPE` value here? If we need it, could we use an `AttributeType` instead, to minimize the `cryptoki-sys` values needed?

Reply:
This could be me not fully understanding how the serialization ends up working in the implementation but the vendor defined type needs to be known along with the data so that the correct value can be specified. I can definitely swap this out with an AttributeType and pull the value out of it.

Reply:
Got it, thanks!